### ISSUE TITLE
Unsupported Camera Format

### ISSUE THREAD

**Issue by Tiim12312** [ayufan-research/camera-streamer#10]
Hi,
I get this when trying to run any of the scripts to start. I'm using the Arducam 16MP.
![image](https://user-images.githubusercontent.com/29235291/187053964-7a518ccd-8153-4706-8639-a354d70aa08a.png)

This is the output of checking the formats available.

![image](https://user-images.githubusercontent.com/29235291/187053985-456b1606-0efb-4763-9d05-393a43b6f869.png)

What is the issue here?
Thanks!

**Comment by ayufan** [comment]
The best is to use preconfigured service: https://github.com/ayufan-research/camera-streamer#use-a-preconfigured-systemd-services. There's one specifically for arducam. You might modify it to suite your needs.

**Comment by Tiim12312** [comment]
Thank you for the fast response. This is the output I get when running it that way.
![image](https://user-images.githubusercontent.com/29235291/187072237-976907a0-422c-4282-88ba-4d164f0b4c79.png)

I have updated the kernel, this is the kernel version.

![image](https://user-images.githubusercontent.com/29235291/187072261-e824126c-13c8-4b6d-8407-fd2289710727.png)

**Comment by ayufan** [comment]
Can you show your `/boot/config.txt`?

**Comment by Tiim12312** [comment]
![image](https://user-images.githubusercontent.com/29235291/187072503-bbe85bf0-0b28-4079-abcb-c3f207045b90.png)

I added this line

**Comment by ayufan** [comment]
OK. I believe you need to change it to `dtoverlay=imx519` or `dtoverlay=imx519,media-controller=1`.

**Comment by Tiim12312** [comment]
I figured out my issue, I did two things. First I didn't realize I needed to install the arducam 16MP drivers from their site.
https://www.arducam.com/docs/cameras-for-raspberry-pi/raspberry-pi-libcamera-guide/how-to-use-arducam-16mp-camera-on-rapberry-pi/

Second I hadn't placed the dtoverlay line under the [all] section of the /boot/config.txt. Not sure which of those things fixed it but seems to be working now! Thank you!!
